src/transformers/pipelines/__init__.py

```diff
@@ -203,14 +203,14 @@
     },
     "conversational": {
         "impl": ConversationalPipeline,
-        "tf": TFAutoModelForCausalLM if is_tf_available() else None,
-        "pt": AutoModelForCausalLM if is_torch_available() else None,
+        "tf": (TFAutoModelForSeq2SeqLM, TFAutoModelForCausalLM) if is_tf_available() else (),
```

Comment:
That's nice! It also looks much closer to our testing design now with `model_classse = (...) if is_tf_available() else ()`

Reply:
Think putting Seq2Seq before CausalLM makes a lot of sense!